spec/acceptance/add_attribute_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe "add_attribute" do
+  before do
+    define_model("Post", title: :string, sequence: :string, new: :boolean)
+
+    FactoryBot.define do
+      factory :post do
+        add_attribute(:title) { "Title" }
+        add_attribute(:sequence) { "Sequence" }
+        add_attribute(:new) { true }
+      end
+    end
+  end
+
+  context "returning attributes for a factory" do
+    subject { FactoryBot.attributes_for(:post) }
+
+    it "assigns default values" do
+      expect(subject[:title]).to eq "Title"
```

Comment:
What do you think about use `have attribute` instead have 3 expects
https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/have-attributes-matcher

Reply:
@gabrielbaldao, thanks for the review!

`#have_attributes` is looking good to me, but I did not find any usage of this method in the code base, so I think I'll leave it as it is for the sake of consistency. =)